core/src/test/scala/unit/kafka/server/ControllerApisTest.scala

```diff
@@ -145,6 +145,14 @@ class ControllerApisTest {
     authorizer
   }
 
+  @Test
+  def testHanldleMetadata(): Unit = {
+    val caught = assertThrows(classOf[ApiException], () => createControllerApis(
+      None, new MockController.Builder().build()).
```

Comment:
I think I might have found an NPE. createControllerApis allows for Option[authorizer], however fails with an NPE if None is passed as an authorizer.

Reply:
Was chasing a phantom. This was not an issue. Incorrect implementation on my part.